tests/manage/monitoring/prometheusmetrics/test_monitoring_defaults.py

```diff
@@ -30,21 +30,25 @@ def test_monitoring_enabled():
     """
     prometheus = PrometheusAPI()
 
-    # ask for values of ceph_pool_stored metric
-    logger.info("Checking that ceph data are provided in OCS monitoring")
-    result = prometheus.query('ceph_pool_stored')
-    msg = "check that we actually received some values for a ceph query"
-    assert len(result) > 0, msg
-    for metric in result:
-        _, value = metric['value']
-        assert_msg = "number of bytes in a pool isn't a positive integer or zero"
-        assert int(value) >= 0, assert_msg
-    # additional check that values makes at least some sense
-    logger.info(
-        "Checking that size of ceph_pool_stored result matches number of pools")
-    ct_pod = pod.get_ceph_tools_pod()
-    ceph_pools = ct_pod.exec_ceph_cmd("ceph osd pool ls")
-    assert len(result) == len(ceph_pools)
+    if not (
+        storagecluster_independent_check()
+        and float(config.ENV_DATA['ocs_version']) < 4.6
+    ):
+        # ask for values of ceph_pool_stored metric
```

Comment:
Shouldn't there be something like:
```
        pytest.skip(
            f"Metrics for external mode is not enabled for OCS "
            f" {float(config.ENV_DATA['ocs_version'])}"
        )
```
?
This way part of the test is not being run but part of the test is run.

Reply:
That was the intention. While we do not have ceph metrics available for external mode, we do have noobaa metrics and that is why I did not apply skip for the entire test.

Reply:
Ok, please add warning log message for case when it is skipped.

Reply:
Done